minor_changes:
  - assible.utils.encrypt now returns `AssibleError` instead of crypt.crypt's `OSError` on Python 3.9
